src/patternfly/components/ContextSelector/context-selector.scss

```diff
@@ -26,6 +26,23 @@
   --pf-c-context-selector__toggle--m-plain--m-text--PaddingRight: var(--pf-global--spacer--sm);
   --pf-c-context-selector__toggle--m-plain--m-text--PaddingLeft: var(--pf-global--spacer--sm);
 
+  // ::before
+  --pf-c-context-selector__toggle--before--BorderWidth: var(--pf-c-context-selector__toggle--BorderWidth); // update value at breaking change
+  --pf-c-context-selector__toggle--before--temp--BorderBottomColor: var(--pf-global--BorderColor--300); // change to --pf-c-context-selector__toggle--before--BorderColor at breaking change
```

Comment:
Why not just call it `--pf-c-context-selector__toggle--before--BorderBottomColor`? Doesn't look like that's used